### ISSUE TITLE
About the location of fnm (Linux)

### ISSUE THREAD

**Issue by matheusAMDS** [Schniz/fnm#1084]
I was trying to uninstall fnm on Linux and the README was mentioning the location `~/.fnm` to be removed but for me it wasn't working, it only worked removing the `~/.local/share/fnm`. Is the latter location really the correct one?

**Comment by Schniz** [comment]
Yes, the docs are outdated. I wonder if there are packages in apt or similar solutions so we can suggest using them instead?

**Comment by Schniz** [comment]
Closing this — it's been open for a while without activity. If you're still seeing this on a recent version, feel free to reopen.

---

*This message was written by AI. Sorry it's not personal, but it's helping me maintain this project. Feel free to disagree with the statement and reopen the issue if needed.*
